lib/reek/smell_detectors/data_clump.rb

```diff
@@ -45,6 +45,10 @@ def self.default_config
           MIN_CLUMP_SIZE_KEY => DEFAULT_MIN_CLUMP_SIZE)
       end
 
+      def self.custom_options
+        Set.new [:max_copies, :min_clump_size]
```

Comment:
Yes, this is duplicated. Using the constants from above in here would require to convert them to symbols because they are strings, which is also not exactly appealing.
Fixes this in this PR would make it completely explode - I intend to fix this in the course of #1123